### PR TITLE
AAP-15176: Fix broken links in controller user guide

### PR DESCRIPTION
**Pull request by hherbly**
This PR fixes broken links in the Controller user guide. See [AAP-15176](https://issues.redhat.com/browse/AAP-15176) for more info.

### CODE REVIEW

File: downstream/modules/platform/proc-controller-create-job-template.adoc [hunk 1]
```diff
@@ -135,7 +135,7 @@ For more information and examples see link:https://docs.ansible.com/ansible/late
 * *Privilege Escalation*: If checked, you enable this playbook to run as an administrator.
 This is the equivalent of passing the `--become` option to the `ansible-playbook` command.
 * *Provisioning Callbacks*: If checked, you enable a host to call back to {ControllerName} through the REST API and invoke the launch of a job from this job template.
-For more information, see link:https://docs.google.com/document/d/1lzlLLP-4eIRPSs4LsHeRoerKHcVWALvGE5uyIXd_rz4/edit[Provisioning Callbacks].
+For more information, see link:https://www.redhat.com/en/blog/satellite-and-ansible-tower-integration-part-2-provisioning-callbacks[Provisioning Callbacks].
```
Comment (hherbly) [inline downstream/modules/platform/proc-controller-create-job-template.adoc:138]: Access denied to the google doc. I made a guess as to the next most appropriate link--please double check.

Comment (EMcWhinn) [reply downstream/modules/platform/proc-controller-create-job-template.adoc:138]: Double checked the Google doc and the link is:
https://docs.ansible.com/automation-controller/latest/html/userguide/job_templates.html#ug-provisioning-callbacks

File: downstream/modules/platform/proc-controller-create-job-template.adoc [hunk 2]
```diff
@@ -26,7 +26,7 @@ Exceptions are noted in the following table.
 - Check: Perform a "dry run" of the playbook and report changes that would be made without actually making them.
 Tasks that do not support check mode are missed and do not not report potential changes.
 
-For more information on job types see the link:http://docs.ansible.com/playbooks_special_topics.html[Playbooks: Special Topics] section of the Ansible documentation.| Yes
+For more information on job types see the link:LINK IS OUT OF DATE--CAN YOU SUGGEST ANOTHER? -HH[Playbooks: Special Topics] section of the Ansible documentation.| Yes
```
[the remark this answers is not in the document]
Comment (EMcWhinn) [reply downstream/modules/platform/proc-controller-create-job-template.adoc:29]: Need to double check, but this is a later version:
https://docs.ansible.com/ansible/2.9/user_guide/playbooks_special_topics.html

File: downstream/modules/platform/proc-controller-create-job-template.adoc (hunk 1 above)
Comment (EMcWhinn) [reply downstream/modules/platform/proc-controller-create-job-template.adoc:138]: I double checked the google doc and the link is:
https://docs.ansible.com/automation-controller/latest/html/userguide/job_templates.html#ug-provisioning-callbacks

**Review by EMcWhinn — APPROVED** [review]
LGTM

File: downstream/modules/platform/proc-controller-create-job-template.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (hherbly) [reply downstream/modules/platform/proc-controller-create-job-template.adoc:29]: At that link there's a banner at the top that says the page is unmaintained. Is there a better option?

[the remark this answers is not in the document]
Comment (hherbly) [reply downstream/modules/platform/proc-controller-create-job-template.adoc]: New link added per Ed's suggestion: https://docs.ansible.com/ansible-core/2.16/playbook_guide/index.html

**Pull request merged**